integration/kubernetes/helm-chart/monitor/source/grafana/dashboard/Alluxio-Prometheus-Grafana-Monitor-New.json

```diff
@@ -0,0 +1,8586 @@
+{
```

Comment:
This is 8.5k lines. Do we need to include this in the codebase or this can actually be generated automatically somehow? Do we need to review its contents?

Reply:
After thinking about it, we can remove this json file  from the codebase and explain the download address of the template in the README.md

Reply:
I suggested to replace the json file to a link to https://grafana.com/grafana/dashboards/